src/main/java/eu/europeana/metis/sandbox/common/exception/RecordValidationException.java

```diff
@@ -14,7 +14,7 @@
 
   public RecordValidationException(String message, String recordId, String nodeId) {
     super(format("Record: %s Node: %s Message: %s ",
-        recordId, nodeId, message), recordId, null);
+        recordId, nodeId, message), recordId, message, null);
     this.nodeId = nodeId;
```

Comment:
Do you anticipate a use for this class variable? I don't think the getter is used anywhere.

Reply:
Not really, I have not found a use for it, but when defining custom exceptions I try to provide relevant info from them, in this case the node id. On the other hand I have no problem removing it, I just did not want my use of it to drive the way it should be defined (if that makes sense)